src/main/resources/application.yml

```
@@ -24,4 +28,14 @@ logging:
\ No newline at end of file
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
+
+cloud:
+  aws:
+    region:
+      static: ${REGION}
+    s3:
+      bucket: ${BUCKET}
+    credentials:
+      access-key: ${AWS_ACCESS_KEY_ID}
+      secret-key: ${AWS_SECRET_ACCESS_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)